<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Job File Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Job File Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="instructionLabel">
      <property name="text">
       <string>Select a job from the list below to transfer its files:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="jobListWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh Jobs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="transferButton">
        <property name="text">
         <string>Transfer Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openSourceButton">
        <property name="text">
         <string>Open Source Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openDestButton">
        <property name="text">
         <string>Open Destination</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>